Create libasicamera2.pc template
--- libasicamera2.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ libasicamera2.pc.in	2016-11-05 23:11:43.247858211 +0000
@@ -0,0 +1,9 @@
+libdir=@LIBDIR@
+includedir=/usr/include
+
+Name: libasicamera
+Description: ZWO ASI camera interface library
+URL: http://astronomy-imaging-camera.com/
+Version: @VERSION@
+Libs: -L${libdir} -lASICamera2
+Cflags: -I${includedir}

